src/components/element-card/element-card.tsx

```diff
@@ -70,11 +76,16 @@ export const ElementCard = ({ element, onClick }: IProps) => {
         event: React.MouseEvent<HTMLButtonElement>
     ) => {
         event.stopPropagation();
+        const newToast = {
+            message: "Element duplicated.",
+            type: EToastTypes.SUCCESS_TOAST,
+        };
 
         if (disableDuplicate) {
             event.preventDefault();
         }
```

Comment:
unrelated: can remove as this will never run; user cant click it as button will be disabled

Reply:
fixed in latest push